monero-outreach-docs/en/quickfacts/quickfacts.md

```diff
@@ -1,12 +1,12 @@
 # MONERO
 
-Monero is a secure, private, untraceable digital currency that is available to all.
+Monero is a secure, private, untraceable digital currency that is available to all. It’s features make it fungible cash.
```

Comment:
*its* features

Reply:
wow, I have been missing some strange things. Thank you erciccione. Done.